### PR TITLE
Update `Create Framework / Unit Testing` page

### PR DESCRIPTION
**Pull request by faissaloux**
Hello 👋🏻 

In this PR I have updated the deprecated `will($this->returnValue())` to `willReturn()`, and the phpunit config file.

### CODE REVIEW

**Review by xabbuh — APPROVED** [review]

**Comment by OskarStark** [comment]
`symfony/phpunit-bridge` does not support PHPUnit 10 now, so the docs should reflect the config for PHPUnit 9.6, which is the current PHPUnit version.

**Comment by faissaloux** [comment]
> `symfony/phpunit-bridge` does not support PHPUnit 10 now, so the docs should reflect the config for PHPUnit 9.6, which is the current PHPUnit version.

So we need to specify the version on the phpunit installation command, because it's installing the current version which is 10 and it conflicts with the rest of the doc.

@OskarStark If accepted I'll update the installation command specifying the version 9.6 instead of updating the doc to phpunit10.

**Comment by OskarStark** [comment]
Sorry I missed this PR :-(

**Comment by faissaloux** [comment]
@OskarStark 

Will `symfony/phpunit-bridge` not support `phpunit10` anytime soon?

Should I update the installation command specifying the version 9.6?

**Comment by OskarStark** [comment]
Please update it, as I don't know when it will be supported

**Comment by faissaloux** [comment]
@OskarStark Done!

File: create_framework/unit_testing.rst [hunk 1]
```diff
@@ -173,7 +173,7 @@ Response::
                 '_route' => 'is_leap_year/{year}',
                 'year' => '2000',
                 '_controller' => [new LeapYearController(), 'index'],
-            ]))
+            ]));
```
Comment (xabbuh) [inline create_framework/unit_testing.rst]:
```suggestion
            ]))
```

Comment (xabbuh) [reply create_framework/unit_testing.rst]: we already have the semicolon on the following line

File: create_framework/unit_testing.rst [hunk 2]
```diff
@@ -12,7 +12,7 @@ using `PHPUnit`_. At first, install PHPUnit as a development dependency:
 
 .. code-block:: terminal
 
-    $ composer require --dev phpunit/phpunit
+    $ composer require --dev phpunit/phpunit:9.6
```
Comment (xabbuh) [inline create_framework/unit_testing.rst]: Shouldn't this be `composer require --dev phpunit/phpunit:^9.6`?

Comment (faissaloux) [reply create_framework/unit_testing.rst:15]: Yes you are right! Thanks

Comment (faissaloux) [reply create_framework/unit_testing.rst]: Yes you are right, thanks!

**Review by xabbuh — APPROVED** [review]

**Comment by javiereguiluz** [comment]
Thanks Faissal, sorry it took us so long to merge this ... and congrats on your first Symfony Docs contribution 🎉

**Pull request merged**